### ISSUE TITLE
Update User Details designs

### ISSUE THREAD

**Issue by jde** [coralproject/design#5]
- [ ] De-emphasize photo (or remove completely if we don't have one.  We only expect a tiny fraction of our users to have photos.
- [ ] Add Notes interface (allow a mod to create a new text note on a User and see the list of notes that were created in the past.)
- [ ] Consider ditching the tab design for a more vertical layout

**Comment by riley** [comment]
The avatar doesn't really need to be big. We could just have a nice placeholder.

I'm fine with re-thinking the tabs thing.

**Comment by egrdina** [comment]
I agree the photo is too big @jde I'm going to look for other ideas of sorting information in the User View as opposed to the tabs. Would be nice to see IF there are notes written about a commenter before a user even clicks into the Notes view. Might also be nice to add a feature "New note" on a user card in the user list view if a note has been added within the last 24 hours.
